src/components/AddBook.js

```diff
@@ -1,18 +1,57 @@
-/* eslint-disable */
+import { useDispatch } from 'react-redux';
+import { v4 } from 'uuid';
+import React, { useState } from 'react';
+import { addBook } from '../redux/books/books';
 
-import React from 'react';
+const AddBook = () => {
+  const [dataTitle, setTitle] = useState('');
+  const [dataAuthor, setAuthor] = useState('');
+  const [category, setCategory] = useState('');
+
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const newBook = {
+      id: v4(), title: dataTitle, author: dataAuthor, category,
+    };
+    setTitle('');
+    setAuthor('');
+    setCategory('');
+    dispatch(addBook(newBook));
+  };
+
+  const add = (e) => {
+    setCategory(e.target.value);
+  };
 
-const AddBook = (props) => {
-  const { title, author, id } = props;
   return (
-    <div className='formContainer'>
-      <form className='form'>
-        <label>Add a new book</label>
-        <input className='input' value={title} type='text' placeholder='Book title'/>
-        <input className='input' value={author} type='text' placeholder='Authors Name'/>
-        <button id={id} type='submit' className='submitBtn'>Submit</button>
+    <div className="formContainer">
+      <form className="form" onSubmit={handleSubmit}>
+        <span className="new-book"> Add New Book</span>
+        <div className="div2">
+          <input className="input" onChange={(e) => setTitle(e.target.value)} value={dataTitle} type="text" placeholder="Title" />
+          <input className="input" onChange={(e) => setAuthor(e.target.value)} value={dataAuthor} type="text" placeholder="Author" />
+          <select name="select" className="select" onChange={add}>
+            <option value="categories" selected disabled>categories</option>
+            <option>Fantasy</option>
+            <option>Emotional</option>
+            <option>Drama</option>
+            <option>Romance</option>
+            <option>Sci-Fi</option>
+            <option>Action</option>
+            <option>Adventure</option>
+            <option>Thriller</option>
+            <option>Animation</option>
+            <option>Crime</option>
+            <option>Horror</option>
+          </select>
+          <button type="submit" className="submitBtn">ADD BOOK</button>
+        </div>
```

Comment:
- [ ] Please ensure that the borders around the inputs are `border: 1px solid #e8e8e8` as specified in the zeplin design.
## Zeplin Design
![image](https://user-images.githubusercontent.com/46839548/202628244-f6be8655-4bda-44a0-b0d5-b1f9fd472697.png)
## Your Implementation
![image](https://user-images.githubusercontent.com/46839548/202628430-291be0f8-8070-4528-a345-48c72d4b4577.png)

Reply:
Thank you for the constructive feedback! 🎉
Issue resolved.